src/effects/builtin/whitenoiseeffect.cpp

```diff
@@ -0,0 +1,62 @@
+#include "whitenoiseeffect.h"
+
+#include <QtDebug>
+#include <QRandomGenerator>
+
+// static
+QString WhiteNoiseEffect::getId() {
+    return "org.mixxx.effects.noise";
+}
+
+// static
+EffectManifestPointer WhiteNoiseEffect::getManifest() {
+    EffectManifestPointer pManifest(new EffectManifest());
+    pManifest->setId(getId());
+    pManifest->setName(QObject::tr("White Noise"));
+    pManifest->setAuthor("The Mixxx Team");
+    pManifest->setVersion("1.0");
+    pManifest->setDescription(QObject::tr("Creates active white noise sound"));
+
+    // This is loudness parameter
+    EffectManifestParameterPointer intensity = pManifest->addParameter();
+    intensity->setId("Loudness");
+    intensity->setName(QObject::tr("Loudness"));
+    intensity->setDescription(QObject::tr("Set the loudness which makes the noise louder or more quiet"));
+    intensity->setControlHint(EffectManifestParameter::ControlHint::KNOB_LOGARITHMIC);
+    intensity->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
+    intensity->setUnitsHint(EffectManifestParameter::UnitsHint::UNKNOWN);
+    intensity->setDefaultLinkType(EffectManifestParameter::LinkType::LINKED);
+    intensity->setMinimum(0);
+    intensity->setDefault(0);
+    intensity->setMaximum(1);
+
+    return pManifest;
+}
+
+WhiteNoiseEffect::WhiteNoiseEffect(EngineEffect* pEffect)
+        : m_pLoudnessParameter(pEffect->getParameterById("Loudness")) {
+}
+
+WhiteNoiseEffect::~WhiteNoiseEffect() {
+}
+
+void WhiteNoiseEffect::processChannel(
+        const ChannelHandle& handle,
+        WhiteNoiseGroupState* pGroupState,
+        const CSAMPLE* pInput,
+        CSAMPLE* pOutput,
+        const mixxx::EngineParameters& bufferParameters,
+        const EffectEnableState enableState,
+        const GroupFeatureState& groupFeatures) {
+    Q_UNUSED(handle);
+    Q_UNUSED(groupFeatures);
+    Q_UNUSED(enableState);
+    Q_UNUSED(pGroupState);
+    
+    CSAMPLE Loudness = m_pLoudnessParameter->value();
```

Comment:
Please don't capitalize variable names, this looks like a class name.

```suggestion
    CSAMPLE loudness = m_pLoudnessParameter->value();
```